src/test/java/software/amazon/event/kafkaconnector/mapping/DefaultEventBridgeMapperTest.java

```diff
@@ -0,0 +1,84 @@
+package software.amazon.event.kafkaconnector.mapping;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.kafka.common.record.TimestampType;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequestEntry;
+import software.amazon.event.kafkaconnector.EventBridgeSinkConfig;
+import software.amazon.event.kafkaconnector.util.MappedSinkRecord;
+
+public class DefaultEventBridgeMapperTest {
+
+  @Test
+  @DisplayName("Mapping without timeMapperClass parameter defined should give time null")
+  public void mapWithDeafultTimeMapper() {
```

Comment:
The purpose of the test was to 1. make sure the default behavoiur is not altered when timeMapper is not overridden and 2. show that it can be overridden using the config param. Will clean up the fieldValue param.

Reply:
Understood. For readability could you use the above outlined approaches to conduct the test? 

Example:

```
var config = new EventBridgeSinkConfig(defaultConfig());
var mapper = new DefaultEventBridgeMapper(config);
var result = mapper.map(List.of(defaultSinkRecord("")));
var putEventsRequestEntry = result.success.get(0).getValue();

// Your assertions
```

The you don't have to cast the results + we don't have to assert the success/errors sizes as we only want to test the time mapping not the behavior of the mapper.